libs/common/src/lib/ai-models/extraction/extraction-modal/llm-configuration.component.ts

```diff
@@ -77,6 +82,7 @@ export class LLMConfigurationComponent implements OnDestroy, OnInit {
         this.valueChange.emit({
           llm: values.customLLM ? this.getLLMConfig(values.llm.generative_model) : undefined,
           merge_pages: values.merge_pages,
+          max_pages_to_merge: values.max_pages_to_merge,
```

Comment:
when `merge_pages` is disabled,  should `undefined` be sent in `max_pages_to_merge` ?

Reply:
yes, that's probably safer, let me fix it